src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,24 @@
+@mixin text-secondary() {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: $secondary-text-color;
+}
+
+@mixin on-mobile {
+  @media (min-width: 488px) {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), media queries should be written for the mobile-first approach. This means defining styles for mobile first and then using min-width for larger devices. The current mixin 'on-mobile' suggests a min-width of 488px, which is not a mobile-first approach. Consider starting with the default styles for mobile and then use mixins for tablet and desktop with appropriate min-width.